interpreter/script/js.ml

```diff
@@ -36,7 +36,10 @@ let spectest = {
   global_f32: 666.6,
   global_f64: 666.6,
   table: new WebAssembly.Table({initial: 10, maximum: 20, element: 'anyfunc'}),
+  table64: new WebAssembly.Table(
+    {initial: 10, maximum: 20, element: 'anyfunc', index: 'i64'}),
```

Comment:
`initial` and `maximum` should be BigInts (`10n` and `20n`) now that #75 has been merged. Same for the memory you added below.

Reply:
Ah, yes, that's a change that I still need to do in V8. Thanks for flagging. I'll first add support for BigInt in V8 and then update this PR.

Reply:
Is there any reason not to land this now?  I was hoping to get some tests added for #75 assuming they could land here and then v8 could pull them in once its is ready?

Reply:
Implementing BigInt support in V8 first would allow to test this PR in V8 before landing it.

Reply:
I'll defer to you here since you are the one doing the work.

However, isn't that backwards from the normal way of doing things.   Don't we normally update the spec + spec tests and then update the implementations once those changes land and get pulled into the testsuite repo?

Reply:
After the change to BigInt landed now in V8, I changed this PR accordingly and verified that it fixes test failures. Feel free to merge if you agree with the change.